src/applications/vaos/components/StatusAlert.jsx

```diff
@@ -43,62 +44,139 @@ export default function StatusAlert({ appointment, facility }) {
-        <div
-          aria-atomic="true"
-          aria-live="assertive"
-          className="vads-u-margin-top--2 vads-u-margin-bottom--0"
-        >
-          <InfoAlert
-            status="error"
-            level={1}
-            headline="We can't access after-visit summaries at this time."
-          >
-            We’re sorry. We’ve run into a problem.
-          </InfoAlert>
+        <InfoAlert backgroundOnly status="info">
+          <p>
+            We’ll try to schedule your appointment in the next 2 business days.
+            Check back here or call your facility for updates. Review your
+            appointments Schedule a new appointment
+          </p>
+        </InfoAlert>
+      );
+    }
+    return (
+      <InfoAlert backgroundOnly status="success">
+        <p>
+          We’ll try to schedule your appointment in the next 2 business days.
+          Check back here or call your facility for updates. Review your
+          appointments Schedule a new appointment
+        </p>
+        <div className="vads-u-margin-y--1">
+          <va-link
+            text="Review your appointments"
+            data-testid="review-appointments-link"
+            href={root.url}
+            onClick={() =>
+              recordEvent({
+                event: `${GA_PREFIX}-view-your-appointments-button-clicked`,
+              })
+            }
+          />
         </div>
+        <div>
+          <va-link
+            text="Schedule a new appointment"
+            data-testid="schedule-appointment-link"
+            onClick={handleClick(history, dispatch, typeOfCare)}
+          />
+        </div>
+      </InfoAlert>
+    );
+  }
+
+  if (featureAfterVisitSummary && !featureAppointmentDetailsRedesign) {
+    const who = canceler.get(appointment?.cancelationReason) || 'Facility';
+
+    if (APPOINTMENT_STATUS.booked === status && isPastAppointment) {
+      return (
+        <>
+          <h2 className="vads-u-margin-bottom--0">After visit summary</h2>
+          <AfterVisitSummary data={appointment} />
+        </>
       );
     }
-    if (!avsLink) {
+
+    if (APPOINTMENT_STATUS.proposed === status) {
+      if (isPastAppointment) {
+        return (
+          <InfoAlert backgroundOnly status="info">
+            <p>
+              We’ll try to schedule your appointment in the next 2 business
+              days. Check back here or call your facility for updates. Review
+              your appointments Schedule a new appointment
+            </p>
+          </InfoAlert>
+        );
+      }
       return (
-        <p className="vads-u-margin--0">
-          An after-visit summary is not available at this time.
-        </p>
+        <InfoAlert backgroundOnly status="success">
+          <p>
+            We’ll try to schedule your appointment in the next 2 business days.
+            Check back here or call your facility for updates. Review your
+            appointments Schedule a new appointment
+          </p>
```

Comment:
Same as above. The last sentence "Review your appointments Schedule a new appointment" can be removed.

Reply:
Done